nebula/ui/components/VPNServerCountry.qml

```diff
@@ -178,24 +181,46 @@ VPNClickableRow {
                 radioButtonLabelText: modelData[1]
                 accessibleName: modelData[1]
                 onClicked: {
+                    if (!isAvailable) {
+                        return;
+                    }
+
                     if (currentServer.whichHop === "singleHopServer") {
                         VPNController.changeServer(code, del._cityName);
                         return stackview.pop();
                     }
 
-                    serversTabs[currentServer.whichHop] = [del._countryCode,  del._cityName, del._localizedCityName] // [countryCode, cityName, localizedCityName]
-                    multiHopStackView.pop()
+                    serversTabs[currentServer.whichHop] = [del._countryCode,  del._cityName, del._localizedCityName]; // [countryCode, cityName, localizedCityName]
+                    multiHopStackView.pop();
                 }
                 height: 54
                 checked: del._countryCode === focusScope.currentServer.countryCode &&  del._cityName === focusScope.currentServer.cityName
                 isHoverable: cityListVisible
-                enabled: cityListVisible
+                enabled: del.isAvailable
```

Comment:
should this be cityListVisible && del.isAvailable? I vaguely remember an issue we had a long time ago where the cities could be clicked when the list was closed.

Reply:
I think we should also restrict `isHoverable` when `!del.isAvailable` unless we are already getting that for free when the button is disabled. I think that ideally there would be no UI changes `onPressed`, `onMouseEnter`, etc when the city is unavailable/disabled.

Reply:
Excellent point! Thank you for catching this potential regression — I added `cityListVisible && del.isAvailable`.

Regarding the UI changes: The `VPNMouseArea` within the `VPNRadioDelegate` now gets disabled as well and should prevent any UI state updates.